### DISCUSSION TITLE
可以改进一下HDR的支持

### DISCUSSION THREAD

**Discussion by flash7777777** [ClassicOldSong/Apollo#51] [Ideas]
之前使用基地版SUNSHINE加黑月亮魔改MOONLIGHT客户端，使用SUNSHINE基地版里内置的虚拟显示器打开显示器的HDR模式，用支持HDR的安卓平板在打开ML里面的HDR选项后串流很完美：既可以看HDR的本地和在线网页视频，又可以串流打HDR游戏。更换了APOLLO的Version v0.2.2-alpha.1以及ARTEMIS后，创建虚拟显示器和不同设备虚拟显示器适配上更方便了，不用再手动一个个建不同分辨率。不过对HDR的支持不是很好，要么ARTEMIS过曝，要么游戏中开HDR颜色变暗。希望下一个版本提高一下对HDR的支持和测试，让不同设备也能无脑连接虚拟显示器打开HDR功能就完美了。

我配置为13600KF / NV4090 / 显示器是用DP连接的1400 亮度 MINILED 32寸显示器，并且主显示器常开HDR。

**Comment by ClassicOldSong** [comment]
Audio/Video设置里勾选 遵循客户端HDR设置 即可
客户端需要配置一下是否使用HDR

**Reply by flash7777777** [reply]
感谢大佬快速回复，经过尝试在APOLLO里面取消无头模式并开启HDR后，客户端可以正确显示HDR了

![image](https://github.com/user-attachments/assets/1b360aab-c26e-4de2-8d31-02162982219e)

**Reply by ClassicOldSong** [reply]
无头模式应该不影响HDR，偶尔遇到过曝的话退出串流重进一次应该就能好了，不用终止串流
